addon/components/bs-form/element.js

```diff
@@ -398,17 +360,17 @@ export default class FormElement extends FormGroup {
    * @type array
    * @protected
    */
-  @defaultValue
-  warnings = null;
+  @arg warnings;
```

Comment:
We should be able to remove this?

Reply:
I thought you prefer that alias if the argument is accessed in the JavaScript file several times. Happy to remove it. Did I got you wrong and you only prefer `@arg` usage if the argument has a default value, which is neither `undefined` nor `null`?

Reply:
> you only prefer @arg usage if the argument has a default value, which is neither undefined nor null?

Yes, that would be my thinking (i.e. don't have duplicated default values). 

Hope this is consistent with any of my previous PRs, but not totally sure 😆